docs/Running.md

```diff
@@ -0,0 +1,136 @@
+# Running the TANF Data Reporting Prototype
+
+This repo has a prototype for a TANF Data Reporting system, where States,
+Territories, and Tribes can upload TANF data and get it validated and stored
+in a database.
+
+## Run this locally
+
+To do local development, you can run this application on your local system.  Here is how
+to get this going:
+* Make sure that [Docker is installed on your system](https://docs.docker.com/install/).
+* Run `./test.sh nodelete` to launch the app using docker-compose.
+* You should be able to access the app on http://localhost:8000/
+
+The app will be running in debug mode and with login.gov disabled.  You will be
+able to log in as `timothy.spencer@gsa.gov` with any password and be an admin in
+the app.  You can stream logs by saying `docker-compose logs -f`.
+
+Be aware that whenever you run `test.sh`, it will entirely delete and recreate the
+database, so you will lose any data you had uploaded to this instance!
+
+You can also run it on your local system without Docker with a few more commands:
+* Make sure that python3 is running on your system.  You can consult one of the
+  many guides on the Internet on how to do this, but here's one that has some good
+  links:  [Properly Installing Python](https://docs.python-guide.org/starting/installation/)
+* Install all the python requirements in a venv: 
+  `python3 -m venv venv ; . venv/bin/activate ; pip install -r requirements.txt`
+* Set up the database:  `NOLOGINGOV=TRUE DEBUG=true ./manage.py migrate`
+* Create a superuser: `NOLOGINGOV=TRUE DEBUG=true ./manage.py createsuperuser`  Add your
+  email address.
+* Run the app!  `NOLOGINGOV=TRUE DEBUG=true ./manage.py runserver`
+* You should be able to access the app on http://localhost:8000/
+
+
+### Testing
+All good applications should be tested.  Here's how you can execute the tests after your
+environment is set up:
+
+`./test.sh`
+
+The python test suites should run and exit cleanly.
+
+## Run this on cloud.gov
+
+You can also deploy this application on [cloud.gov](https://cloud.gov/).
```

Comment:
We might need to add that this needs to be done per environment. We also might want to add a section that future DNS configuration is not yet in here.

Reply:
Given that this app is not public-facing, I'm willing to bet that they don't care about it living on cloud.gov.  But good call on per-environment.  I will have that fixed soon!